.github/workflows/run_tests.yml

```
@@ -228,7 +228,7 @@ jobs:
           cache: 'yarn'
       - name: Install yarn dependencies
         run: yarn install --frozen-lockfile
-      - run: yarn test:coverage
+      - run: yarn test:ci
```

```suggestion
      - run: yarn test:ci
        env:
          GITHUB_ACTIONS: true
```

@ClayBenson94 Is this okay to try to test the reporter?

---

Resolved on a call! (Looks like this actually happens automatically on github runners